features/import.feature

```diff
@@ -179,3 +179,63 @@ Feature: cli-kintone import command
       | Alice  | 10     |
       | Bob    | 20     |
       | Jenny  | 30     |
+
+  Scenario: CliKintoneTest-33 Should import the records successfully with --attachments-dir specified and no attachments fields.
+    Given The csv file "CliKintoneTest-33.csv" with content as below:
+      | Text   | Number |
+      | Alice  | 10     |
+      | Bob    | 20     |
+      | Jenny  | 30     |
+    And Load app ID of app "app_for_import" as env var: "APP_ID"
+    And Load app token of app "app_for_import" with exact permissions "add" as env var: "API_TOKEN_IMPORT"
+    When I run the command with args "record import --base-url $$TEST_KINTONE_BASE_URL --app $APP_ID --api-token $API_TOKEN_IMPORT --attachments-dir ./ --file-path CliKintoneTest-33.csv"
+    Then I should get the exit code is zero
+    And The app "app_for_import" should has records as below:
+      | Text   | Number |
+      | Alice  | 10     |
+      | Bob    | 20     |
+      | Jenny  | 30     |
+
+  Scenario: CliKintoneTest-34 Should import the records successfully with --attachments-dir specified and no attachments fields.
+    Given The csv file "CliKintoneTest-34.csv" with content as below:
+      | Text   | Number | Attachment         |
+      | Alice  | 10     | non_exist_file.txt |
+      | Bob    | 20     | non_exist_file.txt |
+      | Jenny  | 30     | non_exist_file.txt |
+    And Load app ID of app "app_for_import" as env var: "APP_ID"
+    And Load app token of app "app_for_import" with exact permissions "add" as env var: "API_TOKEN_IMPORT"
+    When I run the command with args "record import --base-url $$TEST_KINTONE_BASE_URL --app $APP_ID --api-token $API_TOKEN_IMPORT --attachments-dir ./ --file-path CliKintoneTest-34.csv"
+    Then I should get the exit code is non-zero
+    And The output error message should match with the pattern: "Error: ENOENT: no such file or directory, open 'non_exist_file.txt'"
+
+  Scenario: CliKintoneTest-35 Should import the records successfully with attachments.
+    Given The app "app_for_import" has no records
+    And There is a file "attachments/file1.txt" with content: "123"
+    And There is a file "attachments/file2.txt" with content: "abc"
+    And There is a file "attachments/file3.txt" with content: "!!!"
```

Comment:
This attachment is redundant.
And we only need to cover one attachment in this test case.
Multiple attachments will be covered at `CliKintoneTest-39`